release/_github.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+function invoke_github_api {
+	local curl_response=$(\
+		curl \
+			"${1}" \
+			--data "${2}" \
+			--header "Accept: application/vnd.github+json" \
+			--header "Authorization: Bearer ${LIFERAY_RELEASE_GITHUB_PAT}" \
+			--header "X-GitHub-Api-Version: 2022-11-28" \
+			--include \
+			--max-time 10 \
+			--request POST \
+			--retry 3 \
+			--silent)
+
+	if [ $(echo "${curl_response}" | awk '/^HTTP/{print $2}') -ne 201 ]
+	then
+		lc_log ERROR "Unable to invoke GitHub API:"
+		lc_log ERROR "${curl_response}"
+
+		return "${LIFERAY_COMMON_EXIT_CODE_SKIPPED}"
+	fi
+}
+
+function invoke_github_api_with_response {
+	local curl_response=$(\
+		curl \
+			"${1}" \
+            --data "${2}" \
+			--fail \
+			--header "Accept: application/vnd.github+json" \
+			--header "Authorization: Bearer ${LIFERAY_RELEASE_GITHUB_PAT}" \
+			--header "X-GitHub-Api-Version: 2022-11-28" \
+			--max-time 10 \
+            --include \
+			--request GET \
```

Comment:
@kiwm, the HTTP method is just one more parameter that we can add to this function. :wink: 

Can you try to merge these 2 functions?

Thanks